app/Models/Organization/OrganizationRelationship.php

```diff
@@ -68,19 +69,19 @@ class OrganizationRelationship extends Model
      */
     protected $type;
 
-    /**
-     *
-     * @OA\Property(
-     *     title="relationship_id",
-     *     description="The Organization's relationship_id",
-     *     type="string",
-     *     maxLength=191
-     * )
-     *
-     * @method static OrganizationRelationship whereRelationshipId($value)
-     * @property string $relationship_id
-     */
-    protected $relationship_id;
+    // /**
+    //  *
+    //  * @OA\Property(
+    //  *     title="relationship_id",
+    //  *     description="The Organization's relationship_id",
+    //  *     type="string",
```

Comment:
I think that it should removed because the column: `relationship_id` currently doesn't exist in the `organization_relationships` entity. So, the intention was to remove the  `relationship_id` attribute from the model. Let me remove it.

Reply:
I have created the next PR to remove the `relationship_id` attribute